src/components/Result.js

```diff
@@ -1,25 +1,16 @@
-import React, { Component } from 'react';
-
-class Result extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
-  render() {
-    const { total, next, operation } = this.props;
-    return (
-      <>
-        <h2 className="title">Your Math Calculator</h2>
-        <div className="result-container">
-          <span>{total}</span>
-          <span>{operation}</span>
-          <span>{next}</span>
-        </div>
-      </>
-    );
-  }
-}
+const Result = (props) => {
+  const { total, next, operation } = props;
+  return (
+    <>
+      <h2 className="title">Your Math Calculator</h2>
+      <div className="result-container">
+        <span>{total}</span>
+        <span>{operation}</span>
+        <span>{next}</span>
+      </div>
+    </>
+  );
+};
 // Result.propTypes = {
 //   result: PropTypes.number,
 // };
```

Comment:
- [ ] [Optional] I would suggest you please remove the comment code as it is making the code unclean.